baseline/02_build_tokenizer.py

```diff
@@ -0,0 +1,48 @@
+from tokenizers import (
+    Tokenizer,
+    models,
+    pre_tokenizers,
+    decoders,
+    trainers,
+    processors,
+)
+from datasets import load_dataset
+from functools import partial
+
+from huggingface_hub import HfApi
+import config
+
+
+def batch_iterator(dataset, batch_size=100_000):
+    for sample in dataset.iter(batch_size):
+        texts = sample["en"] + sample["de"]
+        yield texts
+
+
+if __name__ == "__main__":
+    dataset = load_dataset(config.DE_EN_SPLIT_DATASET, split="train+test+validation")
```

Comment:
Should we build the tokenizer using the test and validation set?

Reply:
I am not quite sure, and there is not mention of how to train the tokenizer in the paper. What do you think?